deployment/node-feature-discovery/templates/worker.yaml

```diff
@@ -94,7 +94,10 @@ spec:
             path: "/etc/kubernetes/node-feature-discovery/features.d/"
         - name: nfd-worker-conf
           configMap:
-            name: nfd-worker-conf
+            name: node-feature-discovery
+              items:
+                - key: nfd-worker.conf
```

Comment:
this is no longer aligned with the templates : https://github.com/kubernetes-sigs/node-feature-discovery/blob/2960f63ae5d51bd77015aa4adfe96a90bf753faa/nfd-worker-daemonset.yaml.template#L87

I thought we want to be aligned with the templates as much as possible in the chart.

Reply:
In deed, but I noticed this line https://github.com/adrianchiris/node-feature-discovery/blob/add-helm-chart/deployment/node-feature-discovery/templates/worker.yaml#L97 so it never mounts the config map, bc is a diff name

Reply:
yeah maybe my comment should have been against line https://github.com/adrianchiris/node-feature-discovery/blob/add-helm-chart/deployment/node-feature-discovery/templates/worker.yaml#L97  and not here to make things similar to the templated examples